Dockerfile

```diff
@@ -65,12 +75,10 @@ RUN ln -snf /usr/share/zoneinfo/${TIMEZONE} /etc/localtime && echo ${TIMEZONE} >
     chown -R www-data:www-data /composer
 
 # copy startup script & DB checking script
-COPY .docker/startup.sh /startup.sh
-COPY .docker/service.sh /service.sh
-COPY .docker/self-test.sh /self-test.sh
 COPY .docker/dbtest.php /dbtest.php
+COPY .docker/startup.sh /startup.sh
 
-ENV DATABASE_URL="mysql://kimai:kimai@127.0.0.1:3306/kimai?charset=utf8mb4&serverVersion=5.7.40"
+ENV DATABASE_URL="mysql://kimai:kimai@127.0.0.1:3306/kimai?charset=utf8mb4&serverVersion=8.3"
```

Comment:
why not 8.4?

Reply:
All examples in the documentation point to MySQL 8.3.
It doesn't matter anyway, it just needs to be set during build time and you will overwrite it via ENV.